Make starter_dev fails to build

Composer.lock looks like it's out of date and needs to be updated for the islandora-starter-site
```
docker compose exec -T drupal with-contenv bash -lc 'composer install'
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - chi-teck/drupal-code-generator is locked to version 3.1.0 and an update of this package was not requested.
    - chi-teck/drupal-code-generator 3.1.0 requires php >=8.1.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 2
    - consolidation/log is locked to version 3.0.0 and an update of this package was not requested.
    - consolidation/log 3.0.0 requires php >=8.0.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 3
    - consolidation/robo is locked to version 4.0.6 and an update of this package was not requested.
    - consolidation/robo 4.0.6 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 4
    - consolidation/site-process is locked to version 5.2.0 and an update of this package was not requested.
    - consolidation/site-process 5.2.0 requires php >=8.0.14 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 5
    - drupal/core is locked to version 10.1.2 and an update of this package was not requested.
    - drupal/core 10.1.2 requires composer-runtime-api ^2.1 -> found composer-runtime-api[2.0.0] but it does not match the constraint.
  Problem 6
    - drupal/flysystem is locked to version 2.1.0-rc5 and an update of this package was not requested.
    - drupal/flysystem 2.1.0-rc5 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 7
    - drush/drush is locked to version 12.1.2 and an update of this package was not requested.
    - drush/drush 12.1.2 requires composer-runtime-api ^2.2 -> found composer-runtime-api[2.0.0] but it does not match the constraint.
  Problem 8
    - egulias/email-validator is locked to version 4.0.1 and an update of this package was not requested.
    - egulias/email-validator 4.0.1 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 9
    - grasmash/expander is locked to version 3.0.0 and an update of this package was not requested.
    - grasmash/expander 3.0.0 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 10
    - grasmash/yaml-cli is locked to version 3.1.0 and an update of this package was not requested.
    - grasmash/yaml-cli 3.1.0 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 11
    - laminas/laminas-stdlib is locked to version 3.17.0 and an update of this package was not requested.
    - laminas/laminas-stdlib 3.17.0 requires php ~8.1.0 || ~8.2.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 12
    - league/csv is locked to version 9.9.0 and an update of this package was not requested.
    - league/csv 9.9.0 requires php ^8.1.2 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 13
    - maennchen/zipstream-php is locked to version 3.1.0 and an update of this package was not requested.
    - maennchen/zipstream-php 3.1.0 requires php-64bit ^8.1 -> your php-64bit version (7.4.33) does not satisfy that requirement.
  Problem 14
    - phootwork/collection is locked to version v3.2.2 and an update of this package was not requested.
    - phootwork/collection v3.2.2 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 15
    - phootwork/lang is locked to version v3.2.2 and an update of this package was not requested.
    - phootwork/lang v3.2.2 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 16
    - phpowermove/docblock is locked to version v4.0 and an update of this package was not requested.
    - phpowermove/docblock v4.0 requires php >=8.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 17
    - psr/cache is locked to version 3.0.0 and an update of this package was not requested.
    - psr/cache 3.0.0 requires php >=8.0.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 18
    - psr/log is locked to version 3.0.0 and an update of this package was not requested.
    - psr/log 3.0.0 requires php >=8.0.0 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 19
    - symfony/console is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/console v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 20
    - symfony/dependency-injection is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/dependency-injection v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 21
    - symfony/deprecation-contracts is locked to version v3.3.0 and an update of this package was not requested.
    - symfony/deprecation-contracts v3.3.0 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 22
    - symfony/error-handler is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/error-handler v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 23
    - symfony/event-dispatcher is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/event-dispatcher v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 24
    - symfony/event-dispatcher-contracts is locked to version v3.3.0 and an update of this package was not requested.
    - symfony/event-dispatcher-contracts v3.3.0 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 25
    - symfony/filesystem is locked to version v6.3.1 and an update of this package was not requested.
    - symfony/filesystem v6.3.1 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 26
    - symfony/finder is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/finder v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 27
    - symfony/http-foundation is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/http-foundation v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 28
    - symfony/http-kernel is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/http-kernel v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 29
    - symfony/mime is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/mime v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 30
    - symfony/process is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/process v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 31
    - symfony/routing is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/routing v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 32
    - symfony/serializer is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/serializer v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 33
    - symfony/service-contracts is locked to version v3.3.0 and an update of this package was not requested.
    - symfony/service-contracts v3.3.0 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 34
    - symfony/string is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/string v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 35
    - symfony/translation is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/translation v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 36
    - symfony/translation-contracts is locked to version v3.3.0 and an update of this package was not requested.
    - symfony/translation-contracts v3.3.0 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 37
    - symfony/validator is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/validator v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 38
    - symfony/var-dumper is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/var-dumper v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 39
    - symfony/var-exporter is locked to version v6.3.2 and an update of this package was not requested.
    - symfony/var-exporter v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 40
    - symfony/yaml is locked to version v6.3.3 and an update of this package was not requested.
    - symfony/yaml v6.3.3 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 41
    - doctrine/event-manager is locked to version 2.0.0 and an update of this package was not requested.
    - doctrine/event-manager 2.0.0 requires php ^8.1 -> your php version (7.4.33) does not satisfy that requirement.
  Problem 42
    - symfony/http-foundation v6.3.2 requires php >=8.1 -> your php version (7.4.33) does not satisfy that requirement.
    - drupal/core-recommended 10.1.2 requires symfony/http-foundation ~v6.3.0 -> satisfiable by symfony/http-foundation[v6.3.2].
    - drupal/core-recommended is locked to version 10.1.2 and an update of this package was not requested.

make: *** [starter_dev] Error 2
```

Related to https://github.com/Islandora-Devops/islandora-starter-site/issues/109

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -8,6 +8,8 @@ on:
     - cron: '15 12 * * *'
 jobs:
   make:
+    environment:
+      TERM: xterm-256color
     permissions:
       contents: read
     runs-on: ${{ matrix.os }}
@@ -74,3 +76,12 @@ jobs:
             echo "Failed to bring up site"
             exit 1
           fi
+
+      - name: Notify Slack on nightly test failure
+        if: failure() && github.event_name == 'schedule'
+        run: |-
+          curl -s -o /dev/null -XPOST $SLACK_WEBHOOK_URL -d '{
+            "text": "🚨 Scheduled job failed! Click to view the run: <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|GitHub Actions Run>",
+          }'
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -13,7 +13,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        # TODO: someone with more windows chops please add windows test support
+        # os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-latest]
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
@@ -23,12 +25,14 @@ jobs:
         run: |
           choco install mingw -y
           echo "C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin" >> $GITHUB_PATH
-          
+          cp sample.env .env <-- do not know what windows cp. COPY?
           C:\Program Files\Git\bin\bash.exe ./build/scripts/check-secrets.sh yes
 
       - name: init secrets
         if: matrix.os != 'windows-latest'
-        run: ./build/scripts/check-secrets.sh yes
+        run: |-
+          cp sample.env .env
+          ./build/scripts/check-secrets.sh yes
         shell: bash
 
       - name: make starter

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,0 +1,70 @@
+name: Test
+on:
+  push:
+    paths-ignore:
+      - '**/*.md'
+  schedule:
+    # UTC
+    - cron: '15 12 * * *'
+jobs:
+  make:
+    permissions:
+      contents: read
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set make and secrets for Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          choco install mingw -y
+          echo "C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin" >> $GITHUB_PATH
+          
+          C:\Program Files\Git\bin\bash.exe ./build/scripts/check-secrets.sh yes
+
+      - name: init secrets
+        if: matrix.os != 'windows-latest'
+        run: ./build/scripts/check-secrets.sh yes
+        shell: bash
+
+      - name: make starter
+        run: make starter
+        shell: bash
+
+      - name: check online
+        # TODO: what's a windows curl?
+        if: matrix.os != 'windows-latest'
+        run: |-
+          STATUS=$(curl -k \
+            -w '%{http_code}' -o /dev/null \
+            https://islandora.traefik.me/)
+          echo "Site check returned ${STATUS}"
+          if [ ${STATUS} -ne 200 ]; then
+            echo "Failed to bring up site"
+            exit 1
+          fi
+
+      - name: make build
+        run: make build
+        shell: bash
+
+      - name: make production
+        run: make production
+        shell: bash
+
+      - name: check online
+        # TODO: what's a windows curl?
+        if: matrix.os != 'windows-latest'
+        run: |-
+          STATUS=$(curl -k \
+            -w '%{http_code}' -o /dev/null \
+            https://islandora.traefik.me/)
+          echo "Site check returned ${STATUS}"
+          if [ ${STATUS} -ne 200 ]; then
+            echo "Failed to bring up site"
+            exit 1
+          fi

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -6,10 +6,10 @@ on:
   schedule:
     # UTC
     - cron: '15 12 * * *'
+env:
+  TERM: xterm-256color
 jobs:
   make:
-    environment:
-      TERM: xterm-256color
     permissions:
       contents: read
     runs-on: ${{ matrix.os }}

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -15,7 +15,9 @@ jobs:
       matrix:
         # TODO: someone with more windows chops please add windows test support
         # os: [windows-latest, ubuntu-latest, macos-latest]
-        os: [ubuntu-latest, macos-latest]
+        # TODO: figure out how to install docker in a mac runner :facepalm:
+        # os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest]
     steps:
       - name: Checkout code
         uses: actions/checkout@v4

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -18,7 +18,7 @@ jobs:
         # TODO: someone with more windows chops please add windows test support
         # os: [windows-latest, ubuntu-latest, macos-latest]
         # TODO: keep an eye when macos-14+ (M1) support is available
-        os: [ubuntu-latest, macos-13]
+        os: [ubuntu-latest]
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
@@ -30,10 +30,6 @@ jobs:
           echo "C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin" >> $GITHUB_PATH
           cp sample.env .env <-- do not know what windows cp. COPY?
           C:\Program Files\Git\bin\bash.exe ./build/scripts/check-secrets.sh yes
-
-      - name: Setup docker for mac
-        if: matrix.os == 'macos-13'
-        uses: douglascamata/setup-docker-macos-action@v1-alpha
 
       - name: init secrets
         if: matrix.os != 'windows-latest'

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -17,20 +17,23 @@ jobs:
       matrix:
         # TODO: someone with more windows chops please add windows test support
         # os: [windows-latest, ubuntu-latest, macos-latest]
-        # TODO: figure out how to install docker in a mac runner :facepalm:
-        # os: [ubuntu-latest, macos-latest]
-        os: [ubuntu-latest]
+        # TODO: keep an eye when macos-14+ (M1) support is available
+        os: [ubuntu-latest, macos-13]
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Set make and secrets for Windows
+      - name: Setup make and secrets for Windows
         if: matrix.os == 'windows-latest'
         run: |
           choco install mingw -y
           echo "C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin" >> $GITHUB_PATH
           cp sample.env .env <-- do not know what windows cp. COPY?
           C:\Program Files\Git\bin\bash.exe ./build/scripts/check-secrets.sh yes
+
+      - name: Setup docker for mac
+        if: matrix.os == 'macos-13'
+        uses: douglascamata/setup-docker-macos-action@v1-alpha
 
       - name: init secrets
         if: matrix.os != 'windows-latest'

--- a/Makefile
+++ b/Makefile
@@ -172,7 +172,7 @@ starter_dev: generate-secrets
 	$(MAKE) set-files-owner SRC=$(CURDIR)/codebase ENVIRONMENT=starter_dev
 	docker compose up -d --remove-orphans
 		@echo "Wait for the /var/www/drupal directory to be available"
-	$(MAKE) wait-for-drupal
+	$(MAKE) wait-for-drupal-container
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx /var/www/drupal/ ; su nginx -s /bin/bash -c "composer install"'
 	$(MAKE) starter-finalize ENVIRONMENT=starter_dev
 
@@ -581,17 +581,15 @@ starter-init: generate-secrets
 
 
 .PHONY: starter-finalize
-starter-finalize: wait-for-drupal
+starter-finalize: wait-for-drupal-container
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx . ; echo "Chown Complete"'
 	$(MAKE) drupal-database update-settings-php
 	docker compose exec -T drupal with-contenv bash -lc "drush si -y --existing-config minimal --account-pass '$(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD)'"
 	docker compose exec -T drupal with-contenv bash -lc "drush -l $(SITE) user:role:add fedoraadmin admin"
 	MIGRATE_IMPORT_USER_OPTION=--userid=1 $(MAKE) hydrate
 	docker compose exec -T drupal with-contenv bash -lc 'drush -l $(SITE) migrate:import --userid=1 --tag=islandora'
-	#docker compose exec -T drupal with-contenv bash -lc 'chown -R `id -u`:nginx /var/www/drupal'
-	#docker compose exec -T drupal with-contenv bash -lc 'drush migrate:rollback islandora_defaults_tags,islandora_tags'
 	$(MAKE) login
-
+	$(MAKE) wait-for-drupal-locally
 
 .PHONY: install
 ## Installs drupal site(s) using environment variables.
@@ -680,10 +678,18 @@ fix_views:
 	docker cp scripts/patch_views.sh $$(docker ps --format "{{.Names}}" | grep drupal):/var/www/drupal/patch_views.sh
 	docker compose exec -T drupal with-contenv bash -lc "bash /var/www/drupal/patch_views.sh ; rm /var/www/drupal/patch_views.sh ; drush cr"
 
-.PHONY: wait-for-drupal
-.SILENT: wait-for-drupal
-wait-for-drupal:
+.PHONY: wait-for-drupal-container
+.SILENT: wait-for-drupal-container
+wait-for-drupal-container:
 	while ! docker compose exec -T drupal with-contenv bash -lc 'test -d /var/www/drupal'; do \
 		echo "Waiting for /var/www/drupal directory to be available..."; \
+		sleep 1; \
+	done
+
+.PHONY: wait-for-drupal-locally
+.SILENT: wait-for-drupal-locally
+wait-for-drupal-locally:
+	while ! curl -s -o /dev/null -m 5 https://islandora.traefik.me/ ; do \
+		echo "Waiting for https://islandora.traefik.me to be available..."; \
 		sleep 1; \
 	done

--- a/Makefile
+++ b/Makefile
@@ -157,7 +157,7 @@ starter: generate-secrets
 		docker container run --rm -v $(CURDIR)/codebase:/home/root $(REPOSITORY)/nginx:$(TAG) with-contenv bash -lc 'cd /home/root; composer install'; \
 	fi
 	$(MAKE) set-files-owner SRC=$(CURDIR)/codebase ENVIRONMENT=starter
-	docker compose up -d --remove-orphans
+	$(MAKE) compose-up
 	$(MAKE) starter-finalize ENVIRONMENT=starter
 
 
@@ -170,19 +170,14 @@ starter_dev: generate-secrets
 		docker container run --rm -v $(CURDIR)/codebase:/home/root $(REPOSITORY)/nginx:$(TAG) with-contenv bash -lc 'git clone -b main https://github.com/Islandora-Devops/islandora-starter-site /home/root;'; \
 	fi
 	$(MAKE) set-files-owner SRC=$(CURDIR)/codebase ENVIRONMENT=starter_dev
-	docker compose up -d --remove-orphans
-		@echo "Wait for the /var/www/drupal directory to be available"
-	$(MAKE) wait-for-drupal-container
+	$(MAKE) compose-up
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx /var/www/drupal/ ; su nginx -s /bin/bash -c "composer install"'
 	$(MAKE) starter-finalize ENVIRONMENT=starter_dev
 
 
 .PHONY: production
-production: generate-secrets
-	$(MAKE) download-default-certs
-	$(MAKE) -B docker-compose.yml
-	$(MAKE) pull
-	docker compose up -d --remove-orphans
+production: init
+	$(MAKE) compose-up
 	docker compose exec -T drupal with-contenv bash -lc 'composer install; chown -R nginx:nginx .'
 	$(MAKE) drupal-database update-settings-php
 	docker compose exec -T drupal with-contenv bash -lc "drush si -y --existing-config minimal --account-pass '$(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD)'"
@@ -572,16 +567,18 @@ login:
 	echo "=============================\n"
 
 
-.PHONY: starter-init
-starter-init: generate-secrets
+.PHONY: init
+init: generate-secrets
 	$(MAKE) download-default-certs
 	$(MAKE) -B docker-compose.yml
 	$(MAKE) pull
+
+.PHONY: starter-init
+starter-init: init
 	mkdir -p $(CURDIR)/codebase
 
-
 .PHONY: starter-finalize
-starter-finalize: wait-for-drupal-container
+starter-finalize:
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx . ; echo "Chown Complete"'
 	$(MAKE) drupal-database update-settings-php
 	docker compose exec -T drupal with-contenv bash -lc "drush si -y --existing-config minimal --account-pass '$(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD)'"
@@ -678,9 +675,10 @@ fix_views:
 	docker cp scripts/patch_views.sh $$(docker ps --format "{{.Names}}" | grep drupal):/var/www/drupal/patch_views.sh
 	docker compose exec -T drupal with-contenv bash -lc "bash /var/www/drupal/patch_views.sh ; rm /var/www/drupal/patch_views.sh ; drush cr"
 
-.PHONY: wait-for-drupal-container
-.SILENT: wait-for-drupal-container
-wait-for-drupal-container:
+.PHONY: compose-up
+.SILENT: compose-up
+compose-up:
+	docker compose up -d --remove-orphans
 	while ! docker compose exec -T drupal with-contenv bash -lc 'test -d /var/www/drupal'; do \
 		echo "Waiting for /var/www/drupal directory to be available..."; \
 		sleep 1; \

--- a/Makefile
+++ b/Makefile
@@ -172,10 +172,7 @@ starter_dev: generate-secrets
 	$(MAKE) set-files-owner SRC=$(CURDIR)/codebase ENVIRONMENT=starter_dev
 	docker compose up -d --remove-orphans
 		@echo "Wait for the /var/www/drupal directory to be available"
-	while ! docker compose exec -T drupal with-contenv bash -lc 'test -d /var/www/drupal'; do \
-		echo "Waiting for /var/www/drupal directory to be available..."; \
-		sleep 2; \
-	done
+	$(MAKE) wait-for-drupal
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx /var/www/drupal/ ; su nginx -s /bin/bash -c "composer install"'
 	$(MAKE) starter-finalize ENVIRONMENT=starter_dev
 
@@ -584,7 +581,7 @@ starter-init: generate-secrets
 
 
 .PHONY: starter-finalize
-starter-finalize:
+starter-finalize: wait-for-drupal
 	docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx . ; echo "Chown Complete"'
 	$(MAKE) drupal-database update-settings-php
 	docker compose exec -T drupal with-contenv bash -lc "drush si -y --existing-config minimal --account-pass '$(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD)'"
@@ -682,4 +679,11 @@ fix_masonry:
 fix_views:
 	docker cp scripts/patch_views.sh $$(docker ps --format "{{.Names}}" | grep drupal):/var/www/drupal/patch_views.sh
 	docker compose exec -T drupal with-contenv bash -lc "bash /var/www/drupal/patch_views.sh ; rm /var/www/drupal/patch_views.sh ; drush cr"
-  
+
+.PHONY: wait-for-drupal
+.SILENT: wait-for-drupal
+wait-for-drupal:
+	while ! docker compose exec -T drupal with-contenv bash -lc 'test -d /var/www/drupal'; do \
+		echo "Waiting for /var/www/drupal directory to be available..."; \
+		sleep 1; \
+	done

--- a/sample.env
+++ b/sample.env
@@ -49,7 +49,7 @@ CUSTOM_IMAGE_TAG=latest
 
 # Packagist repo to use when creating a site with make starter
 # Change this if you want to build from a different codebase than the starter site
-CODEBASE_PACKAGE=islandora/islandora-starter-site:1.2.0
+CODEBASE_PACKAGE=islandora/islandora-starter-site:1.6.2
 
 # Includes `traefik` as a service, if false assume we are sharing a traefik
 # from another project.

